src/plugins/barista/tea.rs

```diff
@@ -0,0 +1,447 @@
+use std::fmt;
+
+use rand::distributions::{Distribution, Standard};
+use rand::seq::SliceRandom;
+use rand::Rng;
+
+const CHANCE_OF_TEA_ADJ: f64 = 0.25;
+const CHANCE_OF_VESSEL_SIZE: f64 = 0.90;
+const CHANCE_OF_VESSEL_ADJ: f64 = 0.85;
```

Comment:
nit: `CHANCE_OF_..._ADJECTIVE`, or if you can think of one something that more clearly states "this is the chance that an adjective will precede the tea|vessel"